### ISSUE TITLE
Accessibility Issues on Environments Window: Package View, Package Search Box

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2227]
![image](https://cloud.githubusercontent.com/assets/5177491/23570747/90358688-001a-11e7-8c82-9dd232587b2a.png)

**Comment by eclectir** [comment]
Error details-
MissingProperty - UIA_NameProperty - Element has no accessible name. If a user reaches an element with no name, then often they will have no way of knowing what the element relates to.
https://msdn.microsoft.com/en-us/library/system.windows.automation.automationelement.automationelementinformation.name(v=vs.110).aspx

**Comment by huguesv** [comment]
This has been fixed.
